src/globus_sdk/services/flows/client.py

```diff
@@ -240,25 +241,39 @@ def list_flows(
         List deployed flows
 
         :param filter_role: A role name specifying the minimum permissions required for
-            a flow to be included in the response.
+            a flow to be included in the response. Mutually exclusive with
+            **filter_roles**.
```

Comment:
As a very minor tweak, I think we can call this deprecated here, as a hint to users that they should use `filter_roles` instead:
```suggestion
            **filter_roles**. (deprecated)
```

Reply:
I applied this and it was in the wrong place (filter_roles) but i fixed it 🙃.